base/docs/helpdb.jl

```diff
@@ -4478,42 +4472,40 @@ Obtain the contents of an `IOBuffer` as a string, without copying. Afterwards, t
 takebuf_string
 
 doc"""
-```rst
-..  pipeline(from, to, ...)
+    pipeline(from, to, ...)
 
 Create a pipeline from a data source to a destination. The source and destination can
-be commands, I/O streams, strings, or results of other ``pipeline`` calls. At least one
+be commands, I/O streams, strings, or results of other `pipeline` calls. At least one
 argument must be a command. Strings refer to filenames.
 When called with more than two arguments, they are chained together from left to right.
-For example ``pipeline(a,b,c)`` is equivalent to ``pipeline(pipeline(a,b),c)``. This provides a more
+For example `pipeline(a,b,c)` is equivalent to `pipeline(pipeline(a,b),c)`. This provides a more
 concise way to specify multi-stage pipelines.
 
 **Examples**:
```

Comment:
Hmm running `make -C doc` and checked the html doc, on master it looks like 
![nacsctl-vs-office1](https://cloud.githubusercontent.com/assets/712232/9777953/36fd1a7c-5738-11e5-9c8a-8423e27feb7d.png)

While on this branch it looks like:
![nacsctl-vs-office1](https://cloud.githubusercontent.com/assets/712232/9777958/4071ef4c-5738-11e5-890d-0ef1beb32e5b.png)

The new version doesn't look too bad (honestly I prefer the new version) but this seems to be sth that markdown doesn't fully support (and / or not taken care of by the conversion code) and can introduce inconsistency in the html doc.